dnsserv_test.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	. "gopkg.in/check.v1"
+	"net"
+	"testing"
+)
+
+func Test(t *testing.T) { TestingT(t) }
+
+type DNSNameserverSuite struct{}
+
+var _ = Suite(&DNSNameserverSuite{})
+
+func (s *DNSNameserverSuite) TestUDPConnection(c *C) {
```

Comment:
This testing that we create the expected UDPAddr; it is not a UDP connection test.